### ISSUE TITLE
Monitoring Router using icinga2

### ISSUE THREAD

**Issue by amitlodhiIT** [Icinga/icinga2#7290]
Hi, I am new in icinga2.
I am trying to monitoring the router. For this, I have added in :
 nano /etc/icinga2/conf.d/hosts.conf 
object Host "Router" {
  address = "myRouterIP"
  check_command = "hostalive"
}
The output of icinga ui is : PING OK - Packet loss = 0%, RTA = 0.45 ms
But I want to monitor all the necessary information of router using this. I have heard about check_snmp plugin but I don't know how to use this for getting info.

**Comment by stevie-sy** [comment]
Hi, it depends what you want to Monitor exactly. What are for you "necessary Information"? Uplinks? Performance data? etc.
There exists some plugins listed in the icinga2 docs [https://icinga.com/docs/icinga2/latest/doc/05-service-monitoring/#network-monitoring](url). ;-) 

BTW: This is a bug tracker for the great programmers of icinga. For such questions they have a community Portal: [https://community.icinga.com/](url)

**Comment by mcktr** [comment]
Hi,

this is not a question for the bug tracker, please join https://community.icinga.com/. Create a new topic over there with addional information such as which information are necessary for you and what you want to monitor.

Best regards
Michael
